### ISSUE TITLE
blacklist hash?

### ISSUE THREAD

**Issue by BaronMcG** [OlafvdSpek/xbt#111]
Is it possible to blacklist a hash for a torrent? i got a dmca request from my community tracker and wish to prevent such things in the future. the tracker is open otherwise i can't seem to find a simple way to do this.

**Comment by OlafvdSpek** [comment]
No, that's not possible. You could hard-code the hashes you'd like to blacklist into the code, or create a DB table and read that.
